spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,180 @@
 describe("Gilded Rose", function() {
 
-  it("should do something", function() {
-    update_quality();
-  });
+  describe('normal items (dexterity and Elixir)', () => {
+    it("+5 Dexterity Vest should update quality minus 1 and sell_in minus 1 with sell_In still above zero", function() {
+      items = []
+      items.push(new Item('+5 Dexterity Vest', 10, 20));
 
-});
+      update_quality();
+      expect(items[0]).toEqual(jasmine.objectContaining({
```

Comment:
🆒 😎 ! I didn't know about `jasmine.objectContaining`! Thanks!!

However, is this needed here❓ The item only has three properties, name, sell_in, and quality so partial matching may not be needed.
```
expect(items[0]).toEqual({
        name: '+5 Dexterity Vest',
        sell_in: 9,
        quality: 19
      })
```
Unlike Mocha, Jasmine does the right thing when calling `toEqual` 😄   - The only reason I say it may not be needed is because I'm not sure if jasmine's `toEqual` makes sure that the type is the same. Do you know?